tests/scripts/release-post.test.js

```diff
@@ -194,13 +194,13 @@ test('fetchVersionPolicy(<version>)', (t) => {
   const changelogFixture = path.resolve(__dirname, 'CHANGELOG.fixture.md')
   const changelogLegacyFixture = path.resolve(__dirname, 'CHANGELOG.fixture.legacy.md')
 
-  t.test('finds "Stable" version policy', (t) => {
+  t.test('finds "Current" version policy', (t) => {
     const github = nock('https://raw.githubusercontent.com')
       .get('/nodejs/node/v4.1.0/CHANGELOG.md')
       .replyWithFile(200, changelogFixture)
 
     releasePost.fetchVersionPolicy('4.1.0').then((policy) => {
-      t.equal(policy, 'Stable')
+      t.equal(policy, 'Current')
```

Comment:
This assertion fails because the channel is resolved as "Stable" from the [fixture](https://github.com/nodejs/nodejs.org/blob/04b4e0473c73b8377c3aaa9b2c0a9b2e907afedc/tests/scripts/CHANGELOG.fixture.md)

Maybe it makes sense to create a new fixture faking a changelog for the "Current" channel.